bin/disco_vpc_ui.py

```diff
@@ -107,27 +138,29 @@ def proxy_peerings_command(args):
         vpc_id = None
 
     if args.list_peerings:
-        vpc_map = {vpc.id: vpc for vpc in DiscoVPC.list_vpcs()}
+        vpc_map = {vpc['id']: vpc for vpc in DiscoVPC.list_vpcs()}
         peerings = sorted(
-            DiscoVPC.list_peerings(vpc_id, include_failed=True),
-            key=lambda p: vpc_map.get(p.accepter_vpc_info.vpc_id).tags.get("Name"))
+            DiscoVPCPeerings.list_peerings(vpc_id, include_failed=True),
+            key=lambda p: vpc_map.get(p['AccepterVpcInfo']['VpcId'])['tags'].get("Name"))
```

Comment:
Thanks, good to know. But looks like the post says at the end that it's OK to "use a function without assigning it, e.g: sorted(players, key=lambda player: player.rank)," which is what we are doing here.